lib/react_on_rails/dev/pack_generator.rb

```diff
@@ -21,6 +21,64 @@ def generate(verbose: false)
           puts "❌ Pack generation failed"
           exit 1
         end
+
+        private
+
+        def run_pack_generation(silent: false)
```

Comment:
I don't understand how this relates to the upgrade of ShakaPacker other than this fixes failing tests. This code just doesn't feel right to have in this pull request.